docs/devel/cherry-picks.md

```diff
@@ -64,6 +64,13 @@ hack/cherry_pick_pull.sh upstream/release-3.14 98765
 This will walk you through the steps to propose an automated cherry pick of pull
  #98765 for remote branch `upstream/release-3.14`.
 
+#### Labeling
+
+Ensure your PR contains the label `cherrypick-candidate`.
```

Comment:
After the release-X.Y branch has been created but before X.Y.0:

1. Add cherrypick-candidate and milestone-vX.Y to the PR in master if you want it considered for the next cherrypick batch
2. Once it merges, the release owner will consider it for cherrypicking and will either add cherrypick-approved or remove cherrypick-candidate and the milestone.
3. If approved, once the PR in which the cherrypick appears merges into the release branch, cherrypick-candidate will be removed from the original PR by the bot. (But cherrypick-approved and the milestone will remain.)

I would expect it to work similarly after X.Y.0, except that the PR author would create an individual cherrypick PR using the script.

Every PR targeting the release branch needs to be reviewed, so those PRs don't specifically need cherrypick-candidate. cherrypick-candidate and cherrypick-approved should strictly be for the original PRs to master.